<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WdgThreshold</class>
 <widget class="QWidget" name="WdgThreshold">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>352</width>
    <height>355</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>32767</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Levels</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="chkLogarithmic">
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string>Logarithmic</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>5</number>
     </property>
     <item>
      <widget class="QLabel" name="histview">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>300</height>
        </size>
       </property>
       <property name="lineWidth">
        <number>0</number>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignJustify|Qt::AlignVCenter</set>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <property name="indent">
        <number>-1</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="KisGradientSlider" name="thresholdGradient" native="true">
     <property name="minimumSize">
      <size>
       <width>256</width>
       <height>24</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Threshold &amp;level:</string>
       </property>
       <property name="buddy">
        <cstring>intThreshold</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="intThreshold">
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::PlusMinus</enum>
       </property>
       <property name="maximum">
        <number>255</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KisGradientSlider</class>
   <extends>QWidget</extends>
   <header>KisGradientSlider.h</header>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="local">KisGradientSlider.h</include>
  <include location="local">KisGradientSlider.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
